### PR TITLE
Scenario/C1757

### PR DESCRIPTION
**Pull request by mabalderramav**

### CODE REVIEW

File: IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        /// <summary>
+        /// Veryfis if PinlessPlansTitle label is Displayed.
+        /// </summary>
+        /// <returns>If true, the PinlessPlansTitle label is displayed.</returns>
+        public bool IsLblPinlessPlansTitleDisplayed()
+        {
+            return CommonActions.IsElementDisplayed(LblPinlessPlansTitle);
+        }
+    }
+}
```
Comment (wMercadoJala) [inline IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs]: Add empty line to the end of file

File: IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    /// <summary>
+    /// This class handles the PinlessPlans Page.
+    /// </summary>
+    public class PinlessPlansPage : BasePage
+    {
+        [IDTFindBy(File = "PinlessPlansLocators", Locator = "LblPinlessPlansTitle")]
+        private IWebElement LblPinlessPlansTitle;
+
+        /// <summary>
+        /// Veryfis if PinlessPlansTitle label is Displayed.
```
Comment (virginiaSanabria) [inline IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs]: Wrong "Veryfis"

File: IDT.Pinless/Test/UI/Feature/br_ account.feature [hunk 3]
```diff
@@ -7,39 +7,45 @@
...
-		| PinNumber | (591) 608-9507 |
-
-	Scenario: T39013 - Verify that Print works ok
-		When I search for "(591) 608-9515" number in BR Account
-			And I create Basic plan in "Your Plans" section
-			And I click on Create Account button
-			And I click on Print button
-	# TODO Add assertion about to verify data in pdf.
+# TODO Uncomment these scenarios, when the phone number is unlinked by API is working
+#	Scenario: T39012 - Verify that Basic Pinless creation successful
```
Comment (virginiaSanabria) [inline IDT.Pinless/Test/UI/Feature/br_ account.feature]: Change T39012 by the real testcase's id

File: IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs (hunk 1 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs]: Done

File: IDT.Pinless/Test/UI/Feature/br_ account.feature (hunk 3 above)
Comment (mabalderramav) [reply IDT.Pinless/Test/UI/Feature/br_ account.feature]: Done

File: IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs (hunk 2 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/PinlessPlansPage.cs]: Done

**Pull request merged**